peft/linear_probing.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export TF_CPP_MIN_LOG_LEVEL=2 XLA_PYTHON_CLIENT_PREALLOCATE=false
+
+python3 finetune.py \
+    --model-name="google/timesfm-1.0-200m" \
+    --backend="gpu" \
+    --horizon-len=128 \
+    --context-len=512 \
+    --freq="15min" \
+    --data-path="../datasets/ETT-small/ETTm1.csv" \
+    --num-epochs=100 \
+    --learning-rate=1e-2 \
+    --adam-epsilon=1e-7 \
+    --adam-clip-threshold=1e2 \
+    --early-stop-patience=10 \
+    --datetime-col="date" \
+    --boundaries=1000 46080 57600 \
```

Comment:
why is this 1000 here?

Reply:
My bad, thank you for pointing it out. I will remove this param in the .sh file. The script automatically takes the split as 60-20-20.